server/server.js

```diff
@@ -5,7 +5,7 @@ dotenv.config();
 
 const express = require('express');
 const auctionHandler = require('./controllers/auctions');
-// const db = require('./databas /e/config.js');
+const db = require('./database/config.js');
```

Comment:
I think that we are no longer accessing the database directly from the server file